deps/libjpeg-turbo.gyp

```diff
@@ -117,13 +117,14 @@
         'HAVE_UNISTD_H=1',
         'HAVE_UNSIGNED_CHAR=1',
         'HAVE_UNSIGNED_SHORT=1',
-        'INLINE=inline __attribute__((always_inline))',
         'JPEG_LIB_VERSION=62',
-        'LIBJPEG_TURBO_VERSION="1.4.1"',
+        'LIBJPEG_TURBO_VERSION="1.4.2"',
```

Comment:
Could you also change the BUILD define to match the libjpeg-turbo commit. Looks like it should be 8f1c0a681cd34e8e80ba7b06f356d6080a7172c9.